demos/code_local_assistant/README.md

```diff
@@ -33,7 +33,7 @@ Since we do not want to wait for the code to appear, we need to use smaller mode
 Code completion works in non-streaming, unary mode. Do not use instruct model, there is no chat involved in the process.
 
 Export `Qwen/Qwen2.5-Coder-1.5B`:
-```baconsolesh
+```console
```

Comment:
please also fix: onsole to bash marker for "tree models" command

Reply:
fixed